v2/coordinator-specification.md

```diff
@@ -0,0 +1,691 @@
+        "feeRecipientAddress": "0x78dc5d2d739606d31509c31d654056a45185ecb6",
+        "senderAddress": "0x6ecbe1db9ef729cbe972c83fb886247691fb6beb",
+        "makerAssetAmount": "100000000000000000000",
+        "takerAssetAmount": "200000000000000000000",
+        "makerFee": "1000000000000000000",
+        "takerFee": "1000000000000000000",
+        "expirationTimeSeconds": "1552396423",
+        "salt": "66097384406870180066678463045003379626790660770396923976862707230261946348951",
+        "makerAssetData": "0xf47261b000000000000000000000000034d402f14d58e001d8efbe6585051bf9706aa064",
+        "takerAssetData": "0xf47261b000000000000000000000000025b8fe1de9daf8ba351890744ff28cf7dfa8f5e3",
+        "exchangeAddress": "0x4f833a24e1f95d70f028921e27040ca56e09ab0b"
+      }
+    ],
+    "zeroExTransaction": {
+      "salt": "1495510454898017617137906367124143312199249786920467590762356946976468764317",
+      "signerAddress": "0xe834ec434daba538cd1b9fe1582052b880bd7e63",
+      "data": "0xb4be83d500000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000056bc75e2d6310000000000000000000000000000000000000000000000000000000000000000002a0000000000000000000000000e36ea790bc9d7ab70c55260c66d52b1eca985f84000000000000000000000000000000000000000000000000000000000000000000000000000000000000000078dc5d2d739606d31509c31d654056a45185ecb60000000000000000000000006ecbe1db9ef729cbe972c83fb886247691fb6beb0000000000000000000000000000000000000000000000056bc75e2d6310000000000000000000000000000000000000000000000000000ad78ebc5ac62000000000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000de0b6b3a7640000000000000000000000000000000000000000000000000000000000005c87b0879221cb37dcf690e02b0f9aecf44fcaa5ed9ce99697e86743795fa132596ff597000000000000000000000000000000000000000000000000000000000000018000000000000000000000000000000000000000000000000000000000000001e00000000000000000000000000000000000000000000000000000000000000024f47261b000000000000000000000000034d402f14d58e001d8efbe6585051bf9706aa064000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024f47261b000000000000000000000000025b8fe1de9daf8ba351890744ff28cf7dfa8f5e30000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000421ce8e3c600d933423172b5021158a6be2e818613ff8e762d70ef490c752fd98a626a215f09f169668990414de75a53da221c294a3002f796d004827258b641876e03000000000000000000000000000000000000000000000000000000000000",
+      "verifyingContractAddress": "0x48bacb9266a570d521063ef5dd96e61686dbe788"
+    }
+  }
+}
+```
+
+#### FILL_REQUEST_ACCEPTED
+
+The fill request has been accepted and a signature issued. The corresponding 0x transaction can now be executed on-chain.
+
+[See payload schema](TODO)
+
+```json
+{
+  "type": "FILL_REQUEST_ACCEPTED",
+  "data": {
+    "functionName": "fillOrder",
```

Comment:
This feels a bit redundant to me, since all of this information is included in the transaction. Maybe we can add an on-chain helper method that decodes all of this info out of a transaction.

Reply:
Decided offline to leave it in for now. Once we have an on-chain 0x transaction decoder, we might remove.